finagle/h2/src/main/scala/com/twitter/finagle/buoyant/h2/Stream.scala

```diff
@@ -88,16 +110,12 @@ object Stream {
    * In order to create a stream, we need a mechanism to write to it.
    */
   trait Writer {
-
     /**
      * Write an object to a Stream so that it may be read as a Frame
      * (i.e. onto an underlying transport). The returned future is not
      * satisfied until the frame is written and released.
      */
     def write(frame: Frame): Future[Unit]
-
-    def reset(err: Reset): Unit
-    def close(): Unit
```

Comment:
This method had no purpose.  To close a Stream, simply write a frame with eos=true.